distributions/rhaii/Makefile

```
@@ -0,0 +1,38 @@
+##@ Build
+
+.PHONY: build-frontend
+build-frontend: ## Build the RHAII frontend
+	npm run build
+
+.PHONY: docker-build
+docker-build: ## Build container image
+	cd ../.. && $(CONTAINER_TOOL) build -f distributions/rhaii/Dockerfile -t $(IMG) .
+
+.PHONY: docker-push
+docker-push: ## Push container image to registry
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)